config/README.rst

```diff
@@ -116,6 +116,11 @@ The following arguments are supported.
 --with-pypy
   Enable PyPy support. (Only needed one time as it is stored in .meta.cfg.)
 
+--without-py2
+  The package does not support Python 2, so do not test for it. Also drops
+  support for PyPy2 and Python 3.5 (Only needed one time as it is stored in
+  .meta.cfg.)
```

Comment:
I think I sort of understand why 3.5 is bundled here, but it cries for an explanation in the docs.

Reply:
As the option is now called `--without-legacy-python` it should be much clearer.